gsoc_summary.md

```diff
@@ -0,0 +1,82 @@
+# GSOC 2020 Work product submission.
+
+## Summary of GSoC project with conda-forge (NumFocus)
+
+### Introduction
+  Throughout this Google Summer of code ([GSoC](https://summerofcode.withgoogle.com/)) project I, along with my mentors, aimed to improve the conda-forge "auto-tick, by splitting  bot versions update cron job from the main environment, more specifically, separating the version updates structure to its own microservice. Which in return, improved to a more organic bot ecosystem, not only lessening the burden to the original main process, but also granting the bot team a more versatile way to track the packages version datas, and consecutively it's respective issues.
+
+### What is conda-forge
+  [Conda-forge](https://numfocus.org/project/conda-forge), an umbrella project of [NumFOCUS](https://numfocus.org/), is a [community](https://gitter.im/conda-forge/conda-forge.github.io) effort that provides conda packages for a wide range of software, building and distributing packages, specialized in the hard-to-build or unique packages that often arise in a scientific computing context. The conda-forge ecosystem is fully structured around automation, the "auto-tick" bot plays an important role in the automatic maintenance of conda-forge packages, allowing the maintainers to build packages for different systems architectures, as also keeping their packages up-to-date automatically, and ensuring a high level of quality.
+
+
+### How, it works ?
```

Comment:
I'm not sure how this will looks like when rendered but I recommend removing the space before the question mark.